modular_coyote/eris/code/wielding.dm

```diff
@@ -7,7 +7,7 @@
 	var/wielded_icon //The item state used when it's wielded. Guns are snowflakey and have their own shit for this. This is for non guns.
 	var/force_unwielded = 0 //If you have a specific force for it being unwielded. If for whatever reason you don't want to use the original force of the weapon.
 	var/force_wielded = 0 //If you have a specific force for it being wielded.
-
+	var/wielded_mult = 1
```

Comment:
Would having this always set to 1 by default mean that non-smithed weapons would get no benefit from being wielded?

Reply:
that was the default legit

Reply:
also I believe not, it did apply normally